ci/tasks/build_and_deploy/build_files.sh

```diff
@@ -31,6 +33,10 @@ run_linting(){
   ./get_design_system.sh
   echo "Freezing flask"
 }
+
+export CDA_KEY="$CDA_KEY"
```

Comment:
@joepercival out of interest can we try the script without the export line? my understanding is that variables defined in the main pipeline.yml should already be exported on script startup

Reply:
You're right, it doesn't seem to need it. I've removed them now